### DISCUSSION TITLE
Update to React Select v5

### DISCUSSION THREAD

**Discussion by devRMA** [csandman/chakra-react-select#268] [Ideas]
The new version of react select is available.

https://react-select.com/upgrade

**Comment by csandman** [comment]
This package has been on react-select v5 for a year! v4 of `chakra-react-select` was actually the upgrade to react select v5. Sorry if the numbering confused you, my numbers have not been directly tied to theirs.

https://github.com/csandman/chakra-react-select/blob/main/package.json#L49
